Makefile

```diff
@@ -14,7 +14,8 @@ EXTERNAL_TOOLS=\
 	github.com/client9/misspell/cmd/misspell
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
 
-GO_VERSION_MIN=1.11
+GO_VERSION_MIN=1.12
+GO111MODULE=on
```

Comment:
I think we want to leave this for now as a lot of people are still using the vendor method. I don't really see a reason to force this as it works fine to just build outside your GOPATH with it set to the default of auto.

Reply:
What's the downside of turning this on, other than some additional time required to download the modules when building the first time (as they aren't being referenced from the vendor folder)? It seemed to work just fine for me, so it seems like this is a very easy way to just switch everyone to using modules. And then wouldn't that also enable deleting the vendored copies of everything rather than having to maintain them?

Ultimately, my view is that everyone running builds should get as consistent of a build experience as possible. If, for example, somebody forgets to run `go mod vendor` when a dependency is updated, then somebody running inside the GOPATH would get a different experience than outside the GOPATH. Turning this on here would make that a non-issue.

I also want to acknowledge that I don't really know a lot of what I'm talking about here; I literally knew nothing about the go module system before starting to write #6788. I already had a clone of the repo inside my GOPATH, and this seemed like an easy way to transition people.

Reply:
@joelthompson I am pretty sure that the change you suggest does not affect the `make dev`. 

However, I tried to run `make static-dist dev-ui` but found a first issue which consist in the fact that the go-bindata version that is dragged when I have GO111MODULE not set to off to be quite old and therefore producing an error due to -modtime not being supported ()

However I must also state to @jefferai that the same target `make static-dist dev-ui` does not work with go versions <1.12

This has been tested on both MacOSX and Linux target systems. BTW yarn is also required to build the static-dist + dev-ui target which is not specified anywhere in the requirements.

May be the README could be updated to at least mention some of these issues ...